src/index.js

```diff
@@ -34,12 +35,17 @@ class Document extends YAMLDocument {
 }
 
 function parseAllDocuments(src, options) {
-  return parseCST(src).map(cstDoc => new Document(options).parse(cstDoc))
+  return parseCST(src, {
+    computeLineOffsets: options ? options.computeLineOffsets : false
+  }).map(cstDoc => new Document(options).parse(cstDoc))
```

Comment:
The result of the `cst.map()` operation will not contain the non-iterable properties of `cst`, which means that setting `computeLineOffsets` will not influence the output of this function.

Reply:
Ok, will take another look.